requirements/AUR_CBI_MDL-0120/testing.md

```diff
@@ -0,0 +1,11 @@
+# Testing AUR-CBI-MDL-0120
+
+**1. Steps:**
+
+* Build the project with "make" command
+* Run ./work/binaries/demo file
+
+**2. Expected results:**
+
+* Project should succesfully build
+* The running binary should output logs in a loop with the line: Custom value of custom type: 42
```

Comment:
"[AUR-CBI-MDL-0120]The type of the DataItems stored by the DataStore component shall be user configurable" - while in other tests it is preferred to link to the reference ASN.1, in this case, I would prefer to see ASN.1 with altered (different than in all other tests) DataItem type.

Reply:
OK